kahuna/public/js/util/async.js

```diff
@@ -57,7 +57,7 @@ async.factory("apiPoll", [
   "$q",
   ($q) => {
     const wait = () => new Promise(resolve => {
-      setTimeout(() => resolve(), 500);
+      setTimeout(() => resolve(), 1500);
```

Comment:
this is quite a big increase (x3), how about...
```suggestion
      setTimeout(() => resolve(), 1000);
```
combined with a few more retries (maybe increase 100 to 150 on lines 72 & 73 below - that way it will wait for just as long total as your proposed 100x1500 millis)?